### ISSUE TITLE
L5 vertebra terms

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/mammalian-phenotype-ontology#3729]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. increased volumetric bone mineral density of the L5 vertebra 
2. decreased volumetric bone mineral density of the L5 vertebra 
3. increased compact volumetric bone mineral density of the L5 vertebra
4. decreased compact volumetric bone mineral density of the L5 vertebra
5. increased trabecular volumetric bone mineral density of the L5 vertebra
6. decreased trabecular volumetric bone mineral density of the L5 vertebra
7. increased ultimate load of the L5 vertebra
8. decreased ultimate load of the L5 vertebra
9. abnormal strength of lumbar vertebra

>2. Synonyms (e.g., absent spleen)

1. increased L5 vertebra volumetric bone mineral density
2. decreased L5 vertebra volumetric bone mineral density
3. increased L5 vertebra compact volumetric bone mineral density
4. decreased L5 vertebra compact volumetric bone mineral density
5. increased L5 vertebra trabecular volumetric bone mineral density
6. decreased L5 vertebra trabecular volumetric bone mineral density
7. increased L5 vertebra ultimate load
8. decreased L5 vertebra ultimate load
9. abnormal strength of lumbar vertebra

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. elevation in the qualitative measurement value of mineral content of bone in the L5 vertebra of the spine, measured through the caudo-cranial center of the vertebral body. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
2. reduction in the qualitative measurement value of mineral content of bone in the L5 vertebra of the spine, measured through the caudo-cranial center of the vertebral body. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
3. elevation in the qualitative measurement value of mineral content of compact bone in the L5 vertebra of the spine, measured through the caudo-cranial center of the vertebral body. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
4. reduction in the qualitative measurement value of mineral content of compact bone in the L5 vertebra of the spine, measured through the caudo-cranial center of the vertebral body. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
5. elevation in the qualitative measurement value of mineral content of trabecular bone in the L5 vertebra of the spine, measured through the caudo-cranial center of the vertebral body. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
6. reduction in the qualitative measurement value of mineral content of trabecular bone in the L5 vertebra of the spine, measured through the caudo-cranial center of the vertebral body. This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
7. elevation of the maximum force an L5 vertebra can tolerate before fracturing under axial compression.
8. reduction of the maximum force an L5 vertebra can tolerate before fracturing under axial compression.
9. a change related to the ability of an L5 vertebra to withstand the application of force without yielding or breaking.

PMID:16269906; https://phys.libretexts.org/Bookshelves/Conceptual_Physics/Book%3A_Body_Physics_-_Motion_to_Metabolism_(Davis)/07%3A_Strength_and_Elasticity_of_the_Body/7.01%3A_Strength_of_Human_Bones

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. increased volumetric bone mineral density (MP:0013615)
2. decreased volumetric bone mineral density (MP:0013616)
3. increased volumetric bone mineral density of the L5 vertebra (new term)
4. decreased volumetric bone mineral density of the L5 vertebra (new term)
5. increased volumetric bone mineral density of the L5 vertebra (new term)
6. decreased volumetric bone mineral density of the L5 vertebra (new term)
7. abnormal strength of lumbar vertebra (new term)
8. abnormal strength of lumbar vertebra (new term)
9. abnormal bone strength (new term from issue #3727)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by slaulederkind** [comment]
Preferred term labels 1 and 2 should be:
1. increased total volumetric bone mineral density of the L5 vertebra
2. decreased total volumetric bone mineral density of the L5 vertebra

Suggested parent terms should be:
3. increased total volumetric bone mineral density of the L5 vertebra (new term)
4. decreased total volumetric bone mineral density of the L5 vertebra (new term)
5. increased total volumetric bone mineral density of the L5 vertebra (new term)
6. decreased total volumetric bone mineral density of the L5 vertebra (new term)

**Comment by sbello** [comment]
@slaulederkind How critical is 'L5' to this? Are they just looking at the L5 as a representative of all vertebra or can you see differences between these phenotypes for different vertebra in the same model. I'm hesitant to add phenotypes for specific bones if the exact bone is not an essential part of the phenotype. Would it be enough to have terms for vertebra in general rather than for the L5 specifically?
